spec/card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'poker_hands/card'
+require 'spec_helper'
+
+
```

Comment:
Does Rubocop not complain about this extra whitespace?

Reply:
I totally forgot to add rubocop to this project